main/nic-pip-existing.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string"
+    },
+    "nicName": {
+      "type": "string"
+    },
+    "subnetId": {
+      "type": "string"
+    },
+    "publicIpAddressId": {
+      "type": "string"
+    },
+    "networkSecurityGroupId": {
+      "type": "string"
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Network/networkInterfaces",
+      "apiVersion": "2020-11-01",
+      "name": "[parameters('nicName')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "ipConfigurations": [
+          {
+            "name": "ipconfig1",
+            "properties": {
+              "primary": true,
+              "privateIPAllocationMethod": "Dynamic",
+              "privateIPAddressVersion": "IPv4",
```

Comment:
do AWS support IPV6?

Reply:
In arm template we can configure IPv6, but according to Azure certificate tool we must use IPv4.

Reply:
You mean we should use only IPV4 not both 4 & 5. I mean we should add a option to configure IPV6 as well. May be in next PR

Reply:
OK.